Doc/library/os.rst

```diff
@@ -2121,7 +2121,11 @@ features:
 
    .. audit-event:: os.chmod path,mode,dir_fd os.lchmod
 
-   .. availability:: Unix.
+   .. availability:: Unix, not Linux, FreeBSD >= 1.3, NetBSD >= 1.3, not OpenBSD
+
+   .. note::
+      ``lchmod()`` is not part of POSIX, but Unix implementations may have it
+      if changing the mode of symbolic links is supported.
```

Comment:
I think a note stands out too much, could this be a normal paragraph?

Reply:
I don’t understand why info about lchmod availability does not belong in the lchmod documentation.

The PR description becomes the draft for the commit message, which is used as-is if automerging, or can be edited.
Is this not clear in the devguide?